modules/issue_tracker/ajax/Attachment.php

```diff
@@ -0,0 +1,138 @@
+<?php
```

Comment:
There should not be new ajax scripts added in any pull requests.

Reply:
@driusan adding a new one here is easier than the alternative when there are already other ajax scripts. Otherwise the PR will be worse for reviewers if I convert the other ones to fetch with the new ones.